docs/getting-started-guides/coreos/bare_metal_offline.md

```diff
@@ -176,16 +176,16 @@ This is on the PXE server from the previous section:
 
     rm /etc/httpd/conf.d/welcome.conf
     cd /var/www/html/
-    wget -O kube-register https://github.com/jeffbean/kube-register/releases/download/v0.5/kube-register
+    wget -O kube-register  https://github.com/kelseyhightower/kube-register/releases/download/v0.0.2/kube-register-0.0.2-linux-amd64
```

Comment:
I was testing the new setup with 0.15.0 and read [this mail conversation](https://groups.google.com/forum/#!msg/google-containers/g9MGseIFXTc/QB4663TdDmEJ). The versiont mentioned there fixed it... I guess 0.0.3 would work as well. I tested 0.0.2 and it worked :)